GameData/RealismOverhaul/REWORK/RO_KOSMOS.cfg

```diff
@@ -1000,7 +1000,6 @@
 		name = ModuleEngineConfigs
 		type = ModuleRCS
 		thrustRating = resourceFlowMode = STACK_PRIORITY_SEARCH
```

Comment:
the correct form here is `thrustRating = thrusterPower` and then the `resourceFlowMode` line.